package.json

```diff
@@ -58,7 +58,10 @@
     "lerna-rm-dist": "lerna run rm-dist",
     "lerna-run-test": "lerna run test --stream --no-prefix",
     "lerna-run-test-fork": "lerna run test-fork --stream --no-prefix",
-    "lerna-run-test-with-testrpc": "run-with-testrpc --chainId 1337 --hardfork 'istanbul' --port 8544 --gasLimit 12000000 --defaultBalanceEther 1000 --deterministic 'yarn lerna-run-test'",
+    "lerna-run-test1": "lerna run test1 --stream",
```

Comment:
is this task needed?

Reply:
I run it to run tests faster without compiling, when changing only tests.

Reply:
can you at least give it a descriptive name? thanks